starknet_devnet/blueprints/rpc/structures/payloads.py

```diff
@@ -95,95 +95,163 @@ def config() -> StarknetGeneralConfig:
     return block
 
 
-class RpcInvokeTransaction(TypedDict):
-    """
-    TypedDict for rpc invoke transaction
-    """
+class RpcBroadcastedTxnCommon(TypedDict):
+    """TypedDict for RpcBroadcastedTxnCommon"""
 
-    contract_address: Address
-    entry_point_selector: Felt
-    calldata: List[Felt]
-    # Common
-    transaction_hash: TxnHash
+    type: TxnType
     max_fee: Felt
     version: NumAsHex
     signature: List[Felt]
```

Comment:
Would it make sense to extract this to a separate type `Signature = List[Felt]` since that's how it is in the specifications?